src/assets/icons/ic-bookmark.svg

```diff
@@ -1,10 +1,3 @@
-<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
-<g clip-path="url(#clip0)">
-<path fill-rule="evenodd" clip-rule="evenodd" d="M26.6666 24.2136C26.6666 24.9954 26.0258 25.5473 25.3332 25.5491L25.3333 25.5469V24.2136V5.33329V3.99996H23.9999H9.33325L7.99992 3.99996C7.99992 3.26358 8.59687 2.66663 9.33325 2.66663H25.3333C26.0696 2.66663 26.6666 3.26358 26.6666 3.99996V24.2136ZM6.66659 26.8886L6.66659 6.66663H22.6666V26.8803L16.1439 22.6528C15.2639 22.0824 14.1313 22.0808 13.2496 22.6486L13.9716 23.7696L13.2496 22.6486L6.66659 26.8886ZM7.38856 28.0095L13.9716 23.7696C14.4124 23.4857 14.9787 23.4865 15.4187 23.7717L21.9414 27.9991C22.8285 28.5741 23.9999 27.9374 23.9999 26.8803V6.66663C23.9999 5.93025 23.403 5.33329 22.6666 5.33329H6.66659C5.93021 5.33329 5.33325 5.93025 5.33325 6.66663V26.8886C5.33325 27.944 6.50126 28.581 7.38856 28.0095Z" fill="#E5E5E5"/>
-</g>
-<defs>
-<clipPath id="clip0">
-<rect width="32" height="32" fill="white"/>
-</clipPath>
-</defs>
+<svg width="20" height="28" viewBox="0 0 20 28" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path d="M19 1H1V26L10 20L19 26V1Z" stroke="#52535A" stroke-width="2" />
```

Comment:
Is this file being used by other components?

Reply:
It's used in "ArticleCard" components, to indicate saved article.
It has been updated on the personal page in Figma but hasn't updated on the home page. We can have discussions about this with UI/UX team in the next meeting.